utils/build_swift/driver_arguments.py

```diff
@@ -972,10 +972,6 @@ def create_argument_parser():
         help="the absolute path to libtool. Default is auto detected.",
         type=arguments.type.executable,
         metavar="PATH")
-    parser.add_argument(
-        "--distcc",
```

Comment:
What is this change?

Reply:
I was testing something out locally and mistakenly removed this option. I've added it back.